### DISCUSSION TITLE
Now that this plugin supports ESLint flat config, can it be used with ESLint 9?

### DISCUSSION THREAD

**Discussion by raypatterson** [testing-library/eslint-plugin-testing-library#933] [General]
Relevant Issue: https://github.com/testing-library/eslint-plugin-testing-library/issues/853
Previous Question: https://github.com/testing-library/eslint-plugin-testing-library/issues/853#issuecomment-2138606348

The peer dependencies are limited to ESLint 8.

https://github.com/testing-library/eslint-plugin-testing-library/blob/4dc7caa86b880d7b762a5df23f4ec736a548b502/package.json#L91-L93

**Comment by Belco90** [comment]
The plugin currently supports flat config, but it's not ready for ESLint v9 yet. That will be available when the following PRs are merged:

1. #925 
2. #926 
3. #928

**Reply by raypatterson** [reply]
Ok thank you for the clarification 👍
